webextension/background.js

```diff
@@ -86,7 +86,14 @@ class StudyLifeCycleHandler {
         break;
       }
 
-      browser.cloudstorage.init(browser.runtime.getURL("./skin/clouddownloads.css"), interval).then(
+      browser.cloudstorage.onRecordTelemetry.addListener(
```

Comment:
Presumably we need to remove this listener at some point, too?

Reply:
My understanding that's taken care by EventEmitter  here https://github.com/mozilla/shield-cloudstorage/pull/28/commits/d785003c2c1009f50060646f591a0b59c6fc4942#diff-549919a14832a5cacd7ae883b94a4db1R78